src/Utilities/ToolLocationHelper.cs

```diff
@@ -1167,6 +1167,88 @@ internal static string[] GetApiContractReferences(IEnumerable<ApiContract> apiCo
         }
 
         /// <summary>
+        /// Return the versioned/unversioned SDK content folder path
+        /// </summary>
+        /// <param name="sdkIdentifier">The identifier of the SDK</param>
+        /// <param name="sdkVersion">The verision of the SDK</param>
+        /// <param name="targetPlatformIdentifier">The identifier of the targeted platform</param>
+        /// <param name="targetPlatformMinVersion">The min version of the targeted platform</param>
+        /// <param name="targetPlatformVersion">The version of the targeted platform</param> 
+        /// <param name="folderName">The content folder name under SDK path</param>
+        /// <returns>The SDK content folder path</returns>
+        public static string GetSDKContentFolderPath(
+              string sdkIdentifier,
+              string sdkVersion,
+              string targetPlatformIdentifier,
+              string targetPlatformMinVersion,
+              string targetPlatformVersion,
+              string folderName)
+        {
+            ErrorUtilities.VerifyThrowArgumentLength(sdkIdentifier, "sdkIdentifier");
+            ErrorUtilities.VerifyThrowArgumentLength(sdkVersion, "sdkVersion");
+
+            // Avoid exception in Path.Combine
+            if(folderName == null)
+            {
+                folderName = string.Empty;
+            }
+
+            // If no folder name is input or it isn't UWP SDK, return the root SDK path.
+            if (string.IsNullOrWhiteSpace(folderName) || sdkVersion != "10.0" || !string.Equals(sdkIdentifier, "Windows", StringComparison.OrdinalIgnoreCase))
+            {
+                string sdkLocation = GetPlatformSDKLocation(sdkIdentifier, sdkVersion);
+                return Path.Combine(sdkLocation, folderName);
+            }
+
+            ErrorUtilities.VerifyThrowArgumentLength(targetPlatformIdentifier, "targetPlatformIdentifier");
+            ErrorUtilities.VerifyThrowArgumentLength(targetPlatformVersion, "targetPlatformVersion");
+
+            string sdkContentFolderPath = null;
+            try
+            {
+                TargetPlatformSDK matchingSdk = GetMatchingPlatformSDK(targetPlatformIdentifier, targetPlatformVersion, null, null, null);
+                string platformManifestLocation = null;
+
+                if (matchingSdk != null)
+                {
+                    string platformKey = TargetPlatformSDK.GetSdkKey(targetPlatformIdentifier, targetPlatformVersion);
+
+                    if (!matchingSdk.Platforms.TryGetValue(platformKey, out platformManifestLocation))
+                    {
+                        ErrorUtilities.DebugTraceMessage("GetTargetPlatformReferencesFromManifest", "Target platform location '{0}' did not exist or did not contain Platform.xml", platformManifestLocation);
```

Comment:
Should this "category" (and the one below) be updated? Other places in this file seem to match the method name.